<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registrationView</class>
 <widget class="QWidget" name="registrationView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>341</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="labelAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <item row="2" column="0">
     <widget class="QLabel" name="inserisiUsernameLabel">
      <property name="text">
       <string>Inserisci username</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="Username"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="inserisciPasswordMinimo5CaratteriLabel">
      <property name="text">
       <string>Inserisci password</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="passEdit"/>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;.SF NS Text&quot;;</string>
      </property>
      <property name="text">
       <string>(minimo 5 caratteri)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="confermaPasswordLabel">
      <property name="text">
       <string>Conferma Password</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="confirmPassword"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="nomeLabel">
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="nomeLineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="cognomeLabel">
      <property name="text">
       <string>Cognome</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="cognomeLineEdit"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Sesso:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QRadioButton" name="radioM">
      <property name="text">
       <string>&amp;Maschio</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QRadioButton" name="radioF">
      <property name="text">
       <string>Femmi&amp;na</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QPushButton" name="confirmbtn">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:  rgb(13, 105, 239);
    border-radius: 3px;
	color: white;
}
QPushButton:pressed {
    background-color: rgb(203, 218, 249);
	color: black;
}</string>
      </property>
      <property name="text">
       <string>Conferma</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QPushButton" name="cancelbtn">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:  rgb(13, 105, 239);
    border-radius: 3px;
	color: white;
}
QPushButton:pressed {
    background-color: rgb(203, 218, 249);
	color: black;
}</string>
      </property>
      <property name="text">
       <string>Annulla</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="TypeBox">
      <property name="text">
       <string>Admin</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;.SF NS Text&quot;;</string>
   </property>
   <property name="text">
    <string>Modulo registrazione nuovo utente. 
Compilare tutti i campi della form per registrare un nuovo utente.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
